src/app/loadout-builder/loadout-builder.component.ts

```diff
@@ -268,9 +265,7 @@ function LoadoutBuilderController(
         }
         let bonus = 0;
         let total = 0;
-        // tslint:disable-next-line:prefer-for-of
-        for (let i = 0; i < stats.length; i++) {
-          const stat = stats[i];
+        for (const stat of stats) {
```

Comment:
Lol brave

Reply:
Oh I actually remember this - at the time, babel generated slower code for this than a hand-rolled for loop, so I did this to speed up the old loadout builder. Not sure if it's still true.

Reply:
I looked at the Babel repl and it looks like this is OK now.